packages/blockchain-wallet-v4-frontend/src/modals/Transactions/DownloadTransactions/selectors.js

```diff
@@ -91,30 +91,34 @@ const getEthData = createSelector(
 const getBtcData = createSelector(
   [
     selectors.core.wallet.getWallet,
+    selectors.core.settings.getCurrency,
     selectors.core.data.btc.getTransactionHistory
   ],
-  (wallet, dataR) => {
+  (wallet, currencyR, dataR) => {
+    const currency = currencyR.getOrElse('USD')
     const transform = data => {
-      const transformedData = map(tx => formatTxData(tx, 'BTC'), data)
+      const transformedData = map(tx => formatHaskoinData(tx, 'BTC', currency), data)
```

Comment:
this is totally fine, but if you might not get all the typescripty goodness using the ramda `map`. Might be better off just using good ol `data.map(...)`

Reply:
JS file still so more work needed to get to TS anyway